modules/c/pages/_partials/downloadslist.adoc

```diff
@@ -163,55 +162,89 @@ Enterprise Edition::
 |===
 | Platform | Download | SHA | Debug Symbols
 
-.12+|  Debian
-| {source_url}couchbase-lite-c-enterprise-{our-version}-debian10-x86_64.tar.gz[couchbase-lite-c-enterprise-{our-version}-debian10-x86_64.tar.gz]
-| {source_url}couchbase-lite-c-enterprise-{our-version}-debian10-x86_64.tar.gz.sha256[couchbase-lite-c-enterprise-{our-version}-debian10-x86_64.tar.gz.sha256]
-| {source_url}couchbase-lite-c-enterprise-{our-version}-debian10-x86_64-symbols.tar.gz[couchbase-lite-c-enterprise-{our-version}-debian10-x86_64-symbols.tar.gz]
+.99+|  Debian
```

Comment:
"99" is just "a big number" to colspan, as then we don't have to update this on edits.